dashboard/app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -206,17 +221,25 @@ def silent_takeover(oauth_user, auth_hash)
         # This should never happen if other logic is working correctly, so notify
         Honeybadger.notify(
           error_class: 'Failed to create AuthenticationOption during silent takeover',
-          error_message: "Could not create AuthenticationOption during silent takeover for user with email #{oauth_user.email}"
+          error_message: "Could not create AuthenticationOption during silent takeover for user with email #{lookup_email}"
```

Comment:
Just noticed we're sending PII to Honeybadger here.  I think this is okay - our latest pass on the [Third-party provider list (internal)](https://docs.google.com/spreadsheets/d/1bx0plb-UJ5afkSqU2NkI3XlNc7wmfDN_os8uBbN_cnQ/edit#gid=222345124) mentions we may send PII here in one-off cases.  @poorvasingal and Alex any concerns here?